src/Umbraco.Web.UI.Client/src/common/directives/components/umbgroupsbuilder.directive.js

```diff
@@ -154,8 +154,6 @@
         scope.compositionsDialogModel = {};
         scope.compositionsDialogModel.title = "Compositions";
         scope.compositionsDialogModel.contentType = scope.model;
-        scope.compositionsDialogModel.availableCompositeContentTypes = scope.model.availableCompositeContentTypes;
```

Comment:
Please note this will conflict with:
https://github.com/umbraco/Umbraco-CMS/pull/1008/files#diff-b96ccaa947722f6a466fd094d45d72a1R24
when it is merged in ...